custom_components/solcast_solar/sensor.py

```diff
@@ -298,6 +299,7 @@ class SolcastSensor(CoordinatorEntity, SensorEntity):
 
     _attr_attribution = ATTRIBUTION
     _attr_has_entity_name = True
+    _unrecorded_attributes = frozenset({MATCH_ALL})
```

Comment:
Does this apply to all attributes? If so, I assume that this is a good thing.

Reply:
#goodthing. Stops database bloat. (I recently went on a bloat bash and cut my prod DB from 1.1GiB to 400MiB by cutting a tonne of useless attributes. Things are a bit more 'snappy'. (This was quite involved, but satisfying.)

That one line does the snip for the bulk of sensor attributes, excepting the necessary ones like `device_class`, `state_class`, `unit_of_measurement`, and `friendly_name`, which you want, and don't generally change.

It is implemented for all sensors except the site sensors. I figured that is a miniscule amount of data changes, and actually could be quite nice to forensically dig through when Solcast site changes have been observed in the past. That use case alone made me keep those.

Reply:
(If someone really wants to keep the data for an attribute in LTS they can make a template sensor to grab its value. That t-sensor value would be retained by recorder...)